pkg/k8s/configmap.go

```diff
@@ -1,11 +1,14 @@
 package k8s
 
 import (
+	apiv1alpha1 "github.com/percona/percona-server-mysql-operator/api/v1alpha1"
+	"github.com/percona/percona-server-mysql-operator/pkg/naming"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func ConfigMap(name, namespace, filename, data string) *corev1.ConfigMap {
+func ConfigMap(name, namespace, filename, data string, cr *apiv1alpha1.PerconaServerMySQL) *corev1.ConfigMap {
```

Comment:
Since we are passing the whole cr as an option, do we need to keep the `namespace` as a separate input? We can extract the namespace from cr inside the function.